src/js/views/globals/root.hbs

```diff
@@ -13,5 +13,8 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=1200, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
   </head>
-  <body><div id="root" class="app-root"></div></body>
+  <body>
+    <div style="display:none">{{{ htmlWebpackPlugin.options.faIconSymbols }}}</div>
```

Comment:
@nmajor25 The icons are hidden in a div above the app so that there's no rendering after the fact.
This also means that the app is now rendered in `#root` instead of `body`